Model/Sales/TrackTraceHolder.php

```diff
@@ -232,6 +234,35 @@ public function validateApiKey($apiKey)
         return $this;
     }
 
+    /**
+     * @param Order\Shipment\Track $magentoTrack
+     * @param string               $checkoutData
+     *
+     * @return string
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getLabelDescription($magentoTrack, string $checkoutData): string
+    {
+        $order = $magentoTrack->getShipment()->getOrder();
+
+        $labelDescription = $this->helper->getGeneralConfig(
+            'basic_settings/label_description',
+            $order->getStoreId()
+        );
+
+        if (! $labelDescription) {
+            return '';
+        }
+
+        $deliveryDate     = date('d-m-Y', strtotime($this->convertDeliveryDate($checkoutData)));
+        $labelDescription = str_replace([self::ORDER_NUMBER, self::DELIVERY_DATE], [
```

Comment:
Misschien is het mooier om die arrays wat meer lucht te geven. 
Bijvoorbeeld [self::ORDER_NUMBER, self::DELIVERY_DATE] op zn eigen regen (of meerdere regels)

Reply:
Done